inventories/README.md

```diff
@@ -10,6 +10,33 @@ We recommend to have 3 kinds of inventories:
 - The OpenVSwitch topology inventory, where you will describe the different bridges and ports configuration for OVS: a template is provided (seapath_ovstopology_definition_example.yml)
```

Comment:
In your commit message, missing final point in the commit msg

Reply:
Done